src/NuGet.Core/NuGet.Build.Tasks.Console/NuGet.Build.Tasks.Console.csproj

```diff
@@ -20,6 +19,11 @@
     <UsePublicApiAnalyzer>false</UsePublicApiAnalyzer>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(IsVsixBuild)' == 'true' ">
+    <TargetFrameworks />
+    <TargetFramework>$(NETFXTargetFramework)</TargetFramework>
```

Comment:
FWIW, I don't really see this as an improvement. Nor is it relevant to this PR.

Reply:
Maybe I just didn't read the diff carefully enough, but I thought the outcome is the same, so this change didn't actually change any outcomes. But this change has more lines, plus splits the definition across 20 lines in the file, rather than keeping the IsVsixBuild and not IsVsixBuild on two consecutive lines. 🤷‍♂️

Reply:
Continuing conversation offline.